_posts/2019-06-06-NeuralODEs.markdown

```diff
@@ -0,0 +1,289 @@
+---
+layout: post
+title:  "Neural ODEs"
+date:   2019-06-06 10:00:00 -0400
+categories: neural-nets
+author: luca
+blurb: "Neural ODEs are neural network models which generalize standard layer to layer propagation to continuous depth models. The forward propagation in many neural networks models can be seen as one step of discretation of an ODE; starting from this observation, one can construct and efficiently train models by numerically solving ODEs. On top of furnishing a new whole family of architectures, neural ODEs can be applied to gain memory efficiency in supervised learning tasks, construct a new class of invertible density models and model continuous time-series"
+feedback: true
+---
+
+This guide would not have been possible without the help and feedback from many people. 
+
+Special thanks to Prof. Joan Bruna and his class at NYU, [Mathematics of Deep Learning](https://github.com/joanbruna/MathsDL-spring19), and to Cinjon Resnick, who introduced me to DFL and helped me putting up this guide.
+
+Thank you to Avital Oliver, Matt Johnson, Dougal MacClaurin, David Duvenaud, and Ricky Chen for useful contributions to this guide.
+
+Thank you to Tinghao Li, Chandra Prakash Konkimalla, Manikanta Srikar Yellapragada, Shan-Conrad Wolf, Deshana Desai, Yi Tang, Zhonghui Hu for helping me prepare the notes.
+
+Finally, thank you to all my fellow students who attended the recitations and provided valuable feedback.
+
+
+# Why
+
+Neural ODEs are neural network models which generalize standard layer to layer propagation to continuous depth models. The forward propagation in many neural networks models can be seen as one step of discretation of an ODE; starting from this observation, one can construct and efficiently train models by numerically solving ODEs. On top of furnishing a new whole family of architectures, neural ODEs can be applied to gain memory efficiency in supervised learning tasks, construct a new class of invertible density models and model continuous time-series.
+
+In this curriculum, we will go through all the background topics necessary to understand these models. At the end, you should be able to implement neural ODEs and apply them to different tasks.
```

Comment:
Nice guiding motivation. Maybe put in the blurb at the end?

Reply:
done